### PR TITLE
[CER-3498] Scaling, Endpoints, Storage, Collaboration, Navigation changes

### PR DESCRIPTION
**Pull request by kylegani**

### CODE REVIEW

File: cerebrium/scaling/scaling-apps.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: "Scaling Apps"
+description: "Learn to optimise for cost and performance by scaling out apps"
+---
+
+Cerebrium's scaling system automatically manages computing resources to match app demand. The system handles everything from a few simple requests, to the processing of multiple requests simultaneously, while optimizing for both performance and cost.
+
+## How Autoscaling Works
+
+The scaling system monitors two key metrics to make scaling decisions:
```
Comment (milo157) [inline cerebrium/scaling/scaling-apps.mdx:10]: maybe mention something in this paragraph that its configurable by them based on traffic

File: cerebrium/scaling/batching-concurrency.mdx [hunk 2]
```diff
@@ -0,0 +1,78 @@
+---
+title: "Batching & Concurrency"
+description: "Improve throughput and cost performance with batching & concurrency"
+---
+
+## Understanding Concurrency
+
+Concurrency in Cerebrium allows each instance to process multiple requests simultaneously. The `replica_concurrency` setting in the `cerebrium.toml` file determines how many requests each instance handles in parallel:
```
Comment (milo157) [inline cerebrium/scaling/batching-concurrency.mdx:8]: mention by default his is 1

File: cerebrium/endpoints/async.mdx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Asynchronous Execution (Preview)"
-description: "Execute calls to a Cerebrium app to be run asynchroously"
+title: "Asynchronous Execution"
```
Comment (milo157) [inline cerebrium/endpoints/async.mdx:2]: should we change this to async requests?

File: cerebrium/endpoints/custom-web-servers.mdx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+def health():
+    return "Ok"
+```
+
+Configure this server in `cerebrium.toml` by adding a custom runtime section:
+
+```toml
+[cerebrium.runtime.custom]
+port = 5000
+entrypoint = ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "5000"]
```
Comment (milo157) [inline cerebrium/endpoints/custom-web-servers.mdx:31]: it needs to be main:server based on above implementation

**Review by milo157 — COMMENTED** [review]
Looks good! I think get rid of the "deploying pre-built models" section

File: cerebrium/endpoints/custom-web-servers.mdx (hunk 4 above)
Comment (kylegani) [reply cerebrium/endpoints/custom-web-servers.mdx:31]: Good catch. Changed this instead to be what it is in our examples (And using app)

**Pull request merged**